xds/type/v3/cel.proto

```diff
@@ -23,14 +25,28 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // <https://github.com/google/cel-spec>`_ (CEL) program.
 message CelExpression {
   oneof expr_specifier {
-    option (validate.required) = true;
-
     // Parsed expression in abstract syntax tree (AST) form.
-    google.api.expr.v1alpha1.ParsedExpr parsed_expr = 1;
+    //
+    // Deprecated -- use ``cel_expr_parsed`` field instead.
+    // If ``cel_expr_parsed`` or ``cel_expr_checked`` is set, this field is not used.
+    google.api.expr.v1alpha1.ParsedExpr parsed_expr = 1  [deprecated = true];
 
     // Parsed expression in abstract syntax tree (AST) form that has been successfully type checked.
-    google.api.expr.v1alpha1.CheckedExpr checked_expr = 2;
+    //
+    // Deprecated -- use ``cel_expr_checked`` field instead.
+    // If ``cel_expr_parsed`` or ``cel_expr_checked`` is set, this field is not used.
+    google.api.expr.v1alpha1.CheckedExpr checked_expr = 2  [deprecated = true];
   }
+
+  // Parsed expression in abstract syntax tree (AST) form.
+  //
+  // If ``cel_expr_checked`` is set, this field is not used.
+  cel.expr.ParsedExpr cel_expr_parsed = 3;
```

Comment:
Just curious, why this `cel_expr_parsed` and  `cel_expr_checked` can not be a new oneof like `expr_specifier`

Reply:
Per new proto design policy. I mentioned it in the PR description:

> 1. The new fields `cel_expr_parsed` and `cel_expr_checked` are added outside of `oneof expr_specifier` per updated policy change: https://github.com/envoyproxy/envoy/pull/30851

Reply:
OK Thanks for info!